src/hooks/screen/useChatScreenHook.ts

```diff
@@ -229,7 +254,7 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
 
   React.useEffect(() => {
     initChatData();
-  }, [localDb, chat, selectedChannel]);
+  }, [localDb, chat, otherListener[`chat_${selectedChannel?.id}`], selectedChannel]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The dependencies array of the `useEffect` hook has been updated to include `otherListener[`chat_${selectedChannel?.id}`]`. This is a good practice as it ensures that the effect will re-run whenever any of its dependencies change, keeping the data up-to-date.

```diff
-257:   }, [localDb, chat, otherListener[`chat_${selectedChannel?.id}`], selectedChannel]);
+257:   }, [localDb, chat, otherListener[`chat_${selectedChannel?.id}`], selectedChannel]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->